extensions/quarkus/showcase/pom.xml

```diff
@@ -34,7 +34,7 @@
         <surefire.version>2.22.0</surefire.version>
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>11</java.version>
+        <java.version>17</java.version>
```

Comment:
Reverting this change make the tests pass: 
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Apache MyFaces Core 4.1 - Extensions 4.1.0-SNAPSHOT:
[INFO] 
[INFO] Apache MyFaces Core 4.1 - Extensions ............... SUCCESS [  1.078 s]
[INFO] Apache MyFaces Core 4.1 - Extensions - Quarkus ..... SUCCESS [  0.201 s]
[INFO] Apache MyFaces Core 4.1 - Extensions - Quarkus - Runtime SUCCESS [  2.328 s]
[INFO] Apache MyFaces Core 4.1 - Extensions - Quarkus - Deployment SUCCESS [  1.365 s]
[INFO] Apache MyFaces Core 4.1 - Extensions - Quarkus - Showcase SUCCESS [  9.112 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  14.224 s
[INFO] Finished at: 2024-01-22T15:23:22-05:00
[INFO] ------------------------------------------------------------------------
```

@melloware

Reply:
LGTM!

Reply:
Hmm -- I tried again and it failed?  :(  Not sure what's going on....

Once I get past this, I think I can get some releases out

Reply:
let me take a look